File: C#-Basics/AdvancedTopics/02_PrimeChecker/PrimeChecker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

//Write a method that calculates all prime numbers in given range and returns them as list of integers:

class PrimeChecker
{
    static void Main()
    {
        Console.WriteLine("Insert the number n:");
        int number = int.Parse(Console.ReadLine());
        IsPrime(number);
    }

    static void IsPrime(int number)
    {
        bool result = true;
        if (number == 0 || number == 1)
        {
        result = false;
        }
        else
        {
            for (int a = 2; a <= number / 2; a++)
            {
                if (number % a == 0)
                {
                    result = false;
                }

            }
            Console.WriteLine(result);
        }
    }
}

File: JavaScript-Basics/02-JavaScript-Syntax/js/primeChecker.js
//Write a JavaScript function isPrime(value) that checks if an integer number is prime. Write JS program primeChecker.js
// that checks if a few numbers are prime.
// The result should be printed on the console (true or false) on the console.

    function isPrime(number){
    var divider = 2;
    var maxDivider = Math.sqrt(number);
    var prime = true;
    while (prime && (divider <= maxDivider)) {
        if (number % divider == 0) {
            prime = false;
        }
        divider++;
    }
    console.log(prime);
}
isPrime(7);
isPrime(254);
isPrime(587);